### ISSUE TITLE
Add rfkill and dhcpcd5 to default packages

### ISSUE THREAD

**Issue by robotanarchy** [dderby/debian900#26]
Hello and thanks for the debian900 script!

At least on my n900, I can't connect to any wifi-networks without rfkill unblocking. So I guess it makes sense to install this by default.

Also, I am not sure if there is a dhcp client installed by default, dhcpcd works for me.

**Comment by dderby** [comment]
Thanks for pointing this out.  I don't remember there being a problem with rfkill but I will investigate and fix the default configuration if necessary.  Unfortunately this probably won't be for a few months as this project is on hold due to other commitments.

From what I remember, ConnMan which is installed by default has its own DHCP client.

**Comment by dderby** [comment]
I finally got around to testing this and I was able to use ConnMan to connect to a Wi-Fi network.  It did not require the rfkill package and an IP address was assigned using ConnMan's built-in DHCP client, therefore the default package list does not need to be updated.
